File: app/src/main/java/com/partsystem/partvisitapp/core/database/entity/InvoiceCategoryDetailEntity.java
package com.partsystem.partvisitapp.core.database.entity;

import androidx.room.Entity;
import java.io.Serializable;

@Entity(tableName = "invoice_category_detail_table")
public class InvoiceCategoryDetailEntity implements Serializable {
    public Integer InvoiceCategoryId;

    public Integer UserId;
}

File: app/src/main/java/com/partsystem/partvisitapp/core/database/entity/InvoiceCategoryDetailEntity.kt
package com.partsystem.partvisitapp.core.database.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "InvoiceCategoryDetail")
data class InvoiceCategoryDetailEntity(
    @PrimaryKey val InvoiceCategoryId: Int,
    val UserId: Int,
)
